### ISSUE TITLE
Any plan to support Shell script?

### ISSUE THREAD

**Issue by wangx1ng** [Glavin001/atom-beautify#654]

**Comment by ianremsen** [comment]
+1, it doesn't seem to be a huge burden, definitely sub-C levels of work required.

**Comment by Glavin001** [comment]
Sounds like a duplicate of #451.
FYI, waiting on Pull Request: https://github.com/Glavin001/atom-beautify/pull/1220
